SegmentEditorEngrave/SegmentEditorEngrave.py

```diff
@@ -88,7 +88,10 @@ def test_Engrave1(self):
     slicer.mrmlScene.AddNode(segmentEditorNode)
     segmentEditorWidget.setMRMLSegmentEditorNode(segmentEditorNode)
     segmentEditorWidget.setSegmentationNode(segmentationNode)
-    segmentEditorWidget.setMasterVolumeNode(masterVolumeNode)
+    if slicer.app.majorVersion == 5 and slicer.app.minorVersion >= 1:
```

Comment:
This version check only works if Slicer major version is still 5, but it should not be a problem because we will not have to worry about backward compatibility with the old terms by the time we release Slicer 6.

Reply:
I see. I will take note of this for future use for any future Slicer version (both major/minor).

Reply:
Nowadays I tend to use this pattern, as I find it a bit more readable:

```python
if slicer.app.majorVersion*100+slicer.app.minorVersion < 411:
      sequencesModule = slicer.modules.sequencebrowser
    else:
      sequencesModule = slicer.modules.sequences 
```

Maybe we could add functions like `slicer.app.versionIsGreaterThan(4,11,0)` and `slicer.app.versionIsLessThan(4,11,0)` - but it would take a while before we can use them, because older releases would not have it.

Reply:
The following also seems pretty readable
```python
if (slicer.app.majorVersion, slicer.app.minorVersion) < (4, 11)
  sequencesModule = slicer.modules.sequencebrowser
else:
  sequencesModule = slicer.modules.sequences 
```
Using the packaging module would be more robust though as it follows [PEP440](https://peps.python.org/pep-0440/).
```python
from packaging import version
if version.parse(f"{slicer.app.majorVersion}.{slicer.app.minorVersion}") < version.parse("4.11")
  sequencesModule = slicer.modules.sequencebrowser
else:
  sequencesModule = slicer.modules.sequences 
```

Reply:
> if (slicer.app.majorVersion, slicer.app.minorVersion) < (4, 11)

I like this a lot, thank you! I'll use this from now on.